lib/msf/base/serializer/readable_text.rb

```diff
@@ -684,8 +692,8 @@ def self.dump_notes(mod, indent = '')
         output << "Also known as:\n"
         val.each { |aka| output << "#{indent}#{aka}\n" }
       when 'NOCVE'
-        output << "CVE not available:\n" \
-                  "#{indent}#{val}\n"
+        # Handled by dump_references
```

Comment:
While I understand the desire to deconflict data displayed to the console it seems like moving responsibility to display a `note` to the `dump_references` method sets a poor example for code layout.

This create an inconsistency in processing between `readable_text` processing and `json` processing.

I pose the question, would it be acceptable to simply remove the empty `CVE` processing from `dump_references` altogether and just process the `NOCVE` note more clearly here?

Reply:
That's a legit concern. I think that's a good idea.

Reply:
I do have a secondary concern: only one module (one of mine) currently sets `NOCVE`. Any other module without a CVE will not display a reason. Do we actually care when a module doesn't have a CVE?

I'm inclined either to follow your suggestion or close this PR.

Reply:
I clarified the wording for `NOCVE` in `dump_notes`. Maybe this is a good compromise without looking too duplicated.

Reply:
There has been suggestion that knowing when a module will never have a CVE has value. As a user, I have not seen a reason the references need to tell me when there is not a CVE referenced in the metadata.  For me the old functionality was a cosmetic thing that did not really provide a usability improvement.

Given that I am comfortable with the suggestion of getting rid fo information that may be unnecessary from the output of the console.

Reply:
I think the key takeaway here is that references should be links, not free-form text. That's for notes.